scss/_variables.scss

```diff
@@ -412,7 +412,7 @@ $aspect-ratios: (
 
 // Typography
 //
-// Font, line-height, and color for body text, headings, and more.
+// Font, line-height, color for body text, headings, and more.
```

Comment:
This one is actually correct as-is I think. The first three—font, line-height, and color—apply to the second three—body text, headings, and more.